ci/ray_ci/builder_container.py

```diff
@@ -18,22 +18,24 @@ class PythonVersionInfo(TypedDict):
 
 
 class BuilderContainer(Container):
-    def __init__(self, python_version: str) -> None:
+    def __init__(self, python_version: str, build_type: str = "optimized") -> None:
         super().__init__(
             "manylinux",
             volumes=[f"{os.environ.get('RAYCI_CHECKOUT_DIR')}:/rayci"],
         )
         python_version_info = PYTHON_VERSIONS.get(python_version)
+        self.build_type = build_type
```

Comment:
maybe check if `build_type` is supported, and raise a `ValueError` if it is not?